solana/restaking/programs/restaking/src/lib.rs

```diff
@@ -455,6 +468,7 @@ pub struct Deposit<'info> {
     pub system_program: Program<'info, System>,
     pub rent: Sysvar<'info, Rent>,
 
+    #[account(address = solana_program::sysvar::instructions::ID)]
     ///CHECK:   
```

Comment:
Do we still need this CHECK comment? Same below throughout this PR.

Reply:
I dont think so. I will remove it.

Reply:
Well actually seems like, we need to add `/// CHECK:` if its `AccountInfo` or `UncheckedAccount` irrespective whether we have the validation or not. It seems dumb but thats the way it is.